patches/log.py

```diff
@@ -1,3 +1,4 @@
+import io
```

Comment:
We call `io.StringIO` at line 42: https://github.com/Kaggle/docker-python/blob/43c512adc511ed32608f0cf4629563983b1eb9f3/patches/log.py#L42

This is causing this error: #1324